Windows/MainWindow.GridControls.cs

```diff
@@ -135,24 +189,38 @@ public partial class MainWindow : Window {
             if (Keyboard.FocusedElement is TextBox || !editorIsLoaded) {
                 return;
             }
-            Point mousePos = e.GetPosition(EditorGrid);
-            editorDragSelectStart = mousePos;
-            gridController.GridMouseDown(mousePos);
-            editorMouseDown = true;
+            if (editorIsHoldScrolling) { //Moving with a released wheel and pressing a button
+                scrollEditor_CancelHoldScrolling();
+            } else {
```

Comment:
This should probably be outside of else - check possible problems when transitioning from scrolling to selecting with the same left click.

Reply:
After more testing, this is fine as it is - allowing this logic to fire when hold scrolling ends causes unintuitive behavior with left click, which would require a major refactor to solve.